packages/types/src/options.ts

```diff
@@ -78,7 +78,14 @@ export interface Options {
   /** A global sample rate to apply to all events (0 - 1). */
   sampleRate?: number;
 
-  /** A global sample rate to apply to all transactions (0 - 1). */
+  /**
+   * Sample rate to determine transaction/span sampling.
```

Comment:
We don't sample on the span level.

```suggestion
   * Sample rate to determine transaction sampling.
```

Which reveals the awkward name, "traces..."

```suggestion
   * Sample rate to determine trace sampling.
```

If we think that either this transaction is part of an existing trace or is the origin of a new trace, than calling it "trace sample rate" is correct. That is because what is sampled in the whole trace, and not individual transactions.

Reply:
Right now, a new transaction === a new trace, so in that sense it makes sense.